Resources/Prototypes/DeltaV/Entities/Objects/Misc/identification_cards.yml

```
@@ -0,0 +1,38 @@
+- type: entity
+  parent: IDCardStandard
+  id: CJIDCard
```

```suggestion
  id: ChiefJusticeIDCard
```
This doesn't follow the standard, so should be ChiefJusticeIDCard